data/blog/configuring-emacs-mdx.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Configuring Emacs for MDX files
+date: 2023-02-03
+authors: ['xe']
```

Comment:
Almost forgot. Can you add a `summary` here?

Reply:
Done in https://github.com/tailscale-dev/tailscale-dev/pull/46/commits/77a7bda33158bb8b87ff6b8860e6e00eb0c4c2d2